src/lib/classes/lsp7-digital-asset.ts

```diff
@@ -101,15 +101,16 @@ export class LSP7DigitalAsset {
       digitalAssetConfiguration?.byteCode
     );
 
+    const signerIsUniversalProfile$ = isSignerUniversalProfile$(this.signer);
```

Comment:
FMI : why is there $ at the end of the name of the variable ?

Reply:
this is an rxjs standard which indicated it returns an observable and not the actual value